### PR TITLE
gh-82530: Consolidate comments in email example

### PR DESCRIPTION
**Pull request by slateny**
* Issue: gh-82530

### CODE REVIEW

File: Doc/includes/email-read-alternative.py [hunk 1]
```diff
@@ -9,6 +9,9 @@
 from email.parser import BytesParser
 
 # An imaginary module that would make this work and be safe.
+# The magic_html_parser has to rewrite the href="cid:...." attributes to
+# point to the filenames in partfiles.  It also has to do a safety-sanitize
+# of the html.  It could be written using html.parser.
 from imaginary import magic_html_parser
```
Comment (iritkatriel) [inline Doc/includes/email-read-alternative.py]: We could also rename imaginary to imaginary_module and then it won't look like a real module name.

**Review by kumiDa — APPROVED** [review]
The requested changes are made to the PR, looks good to me.
Now it only requires a Core-member review.

**Comment by terryjreedy** [comment]
See https://github.com/python/cpython/issues/82530#issuecomment-1251793609 for close reason and alternate solution.  slateny, please say if you are doing replacement PR.

**Comment by slateny** [comment]
Sure, I can do the replacement PR.

**Pull request closed without merging**